### DISCUSSION TITLE
Status Auto angesteckt

### DISCUSSION THREAD

**Discussion by fasutec** [goecharger/go-eCharger-API-v2#167] [Q&A]
Hallo liebe Community,
hab eine kleine Frage zur Status-Abfrage über HTTP api:

**Wie kann ich feststellen, ob das Auto angesteckt ist?**

Mittels `.../api/status?filter=modelStatus` bekomm ich z.B. 17 (NotChargingBecauseFallbackAwattar) obwohl das Auto gar nicht mehr angesteckt ist.

Grüße
Mike

**Comment by abraxas87** [comment]
Hallo,

dafür kannst du den API-Key "car" abfragen.

**Reply by fasutec** [reply]
Super danke. Den hab ich voll übersehen.

Noch eine Frage:
Wie kann ich unterscheiden ob die Ladung vom Auto erfolgreich beendet wurde (weil voll) oder ob er nur eine Ladepause macht wegen keinem PV Überschuss.
Es geht mir darum, ob ich dann den heizstab starten darf oder nicht.

In beiden Fällen habe ich modelStatus=17 und car=4

**Reply by abraxas87** [reply]
Ich denke das bekommst du so über die API nicht raus.
Aber du weist ja ob du PV Überschuss hast und das Auto trozdem nicht lädt, oder?
